_lessons/dbt-fundamentals.html

```diff
@@ -160,6 +160,19 @@
 <img src="/ui/img{{page.id}}/model.png" style="width: 50%;">
 ]
 
+---
+## demo
+
+<br>
+<br>
+.center[
+<img src="/ui/img/dbt-standalone-new.png" style="width: 20%;">
+]
+
+???
+Demo Time:
+- jump into cloud and talk about model files, folder structure, what a dbt run command does, etc.
+
```

Comment:
suggestion: Can you add more detail on what to demo here? Since this is after models, I would suggest a model to pull up. Show the SQL select statement. Do a dbt run and show how each model corresponds to an object in the warehouse

Reply:
done!